client/extensions/zoninator/state/locks/actions.js

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	ZONINATOR_REQUEST_LOCK,
+	ZONINATOR_REQUEST_LOCK_ERROR,
+	ZONINATOR_UPDATE_LOCK,
+} from '../action-types';
+
+/**
+ * Returns an action object to indicate that a zone lock should be updated
+ *
+ * @param  {Number}  siteId          Site ID
+ * @param  {Number}  zoneId          Zone ID
+ * @param  {Date}    expires         Expiration date
+ * @param  {Number}  maxLockPeriod   Maximum number of seconds to extend the lock to
+ * @param  {Boolean} reset           Set to true to reset a lock
+ * @return {Action}                  Action object
+ */
+export const updateLock = ( siteId, zoneId, expires, maxLockPeriod, reset ) => ( {
+	type: ZONINATOR_UPDATE_LOCK,
+	siteId,
+	zoneId,
+	expires,
+	maxLockPeriod,
+	reset,
+} );
+
+/**
+ * Returns an action object to indicate that a zone lock should be requested
+ *
+ * @param  {Number}  siteId  Site ID
+ * @param  {Number}  zoneId  Zone ID
+ * @param  {Boolean} reset   Set to true to reset a lock
```

Comment:
Is this a typo? Or can you actually "reset" a lock when you're requesting it? If that is the case, then I don't see any handling of this in the reducer.

Reply:
Well, the idea was to pass that value to the following action on success in case the timer was to be reset, but I see that there's no point in doing that as I can just change the reducer to work with `ZONINATOR_REQUEST_LOCK`.